### PR TITLE
fix: Preview item when user logged in as Item content creator

### PR DESCRIPTION
**Pull request by chinnu-focaloid**
Related Ticket - https://oat-sa.atlassian.net/browse/ADF-576

Description
User with Item Content Creator role gets logged out at Preview attempt for item with asset in it.

### CODE REVIEW

**Review by shpran — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by boajer — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by shpran — CHANGES_REQUESTED** [review]
Please, don't update old migration - it can be executed already.
Move these line into new migration

**Comment by boajer** [comment]
Please ensure all external dependencies updated to be contain neeeded code, e.g. https://github.com/oat-sa/extension-tao-item/pull/492 needs to be used for `TaoItemsRoles::ITEM_CONTENT_CREATOR`

File: migrations/Version202108021320306771_taoQtiTestPreviewer.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        SetRolesAccess::CONFIG_RULES => [            
+            TaoItemsRoles::ITEM_CONTENT_CREATOR => [
+                ['ext' => 'taoQtiTestPreviewer', 'mod' => 'Previewer', 'act' => 'asset'],
+            ],
+        ],
+    ];
+
+    public function getDescription(): string
+    {
+        return 'Assign permissions to Item Previewer role';
```
Comment (shpran) [inline migrations/Version202108021320306771_taoQtiTestPreviewer.php:42]: You add this permission to Item Content Creator, not for Previewer

**Review by shpran — APPROVED** [review]
LGTM

File: migrations/Version202108021320306771_taoQtiTestPreviewer.php (hunk 1 above)
Comment (chinnu-focaloid) [reply migrations/Version202108021320306771_taoQtiTestPreviewer.php]: Done

**Comment by Natallia-Radkevich** [comment]
QA verified on http://test-chinnufrancis.playground.kitchen.it.taocloud.org:48831/ Sprint 158

**Pull request merged**